### PR TITLE
Prometheus integration with byron proxy

### PR DESCRIPTION
**Pull request by disassembler**

### CODE REVIEW

**Comment by disassembler** [comment]
bors ping

**Comment by disassembler** [comment]
bors try

**Comment by 365andreas** [comment]
Wait until we re-export `contramap` from `contra-tracer`.

File: byron-proxy/cfg/logging.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  - KatipBK
+
+# if wanted, the GUI is listening on this port:
+# hasGUI: 18321
+
+# if wanted, the EKG interface is listening on this port:
+hasEKG: 12789
+
+# if wanted, the Prometheus interface is using this port:
+hasPrometheus: 12799
```
Comment (avieth) [inline byron-proxy/cfg/logging.yaml]: Will the program crash if prometheus or ekg is not available at the prescribed port? Can the host be given as well?

Comment (disassembler) [reply byron-proxy/cfg/logging.yaml]: @avieth that's not how prometheus works. This launches a webserver on that port with the metrics published to `/metrics` on that port. I've just tested and it's working as expected!

Comment (disassembler) [reply byron-proxy/cfg/logging.yaml]: EKG is the same, it's a web interface to view the same statistics. This just gives us an easier way to ingest the metrics into grafana.

**Pull request merged**